.github/workflows/release-publish.yml

```
@@ -59,39 +59,18 @@ jobs:
           fi
-          VERSION=$(node -p "require('./packages/$DIR/package.json').version")
-          echo "dir=$DIR" >> $GITHUB_OUTPUT
-          echo "name=$PKG" >> $GITHUB_OUTPUT
-          echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+      - name: Summarize
+        env:
+          EVENT: ${{ github.event_name }}
+          PKG: ${{ steps.pkg.outputs.name }}
+          VERSION: ${{ steps.pkg.outputs.version }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:6:6: Double quote to prevent globbing and word splitting

(shellcheck)